### ISSUE TITLE
Extension on Other filetypes

### ISSUE THREAD

**Issue by CraigRegester** [intersystems/git-source-control#93]
See issue #92 for examples

HL7 schemas with `git-source-control` is outputting with a `.CLS` extension. `InterSystems ObjectScript extension` (see output above), outputs with `.hl7` like I would expect/prefer. 

Same scenario also happens with `LUT `files. I have both LUT and HL7 in my Mappings in Settings but it makes no difference - though does folder them like I want/specify in the mapping. Can we respect the original extensions and just use a .gitattributes file to instruct 'make them pretty' in GitHub (may not be necessary as they are XML)

```
*.cls linguist-language=ObjectScript
*.mac linguist-language=ObjectScript
*.int linguist-language=ObjectScript
*.inc linguist-language=ObjectScript
*.inc linguist-language=ObjectScript
*.csp linguist-language=Html

*.sh text eol=lf
*.cls text eol=lf
*.mac text eol=lf
*.int text eol=lf
Dockerfil* text eol=lf
```

**Comment by isc-tleavitt** [comment]
@isc-svelury we should just always use the extension from the internal name.

**Comment by isc-tleavitt** [comment]
Fixed in #100. Will need to get a new release out soon...

**Comment by CraigRegester** [comment]
@isc-tleavitt echoing need for new release as last one was over 30 days now and I think you guys knocked out a ton of things (that would have been showstoppers for us) so excited to see if I can have others on our team start using this!

Appreciate you both, keep up the good work and thanks for all the efforts.

**Comment by isc-tleavitt** [comment]
@CraigRegester done - published 1.2.0
